### PR TITLE
Update switch command

### PR DESCRIPTION
**Pull request by nweiyue**
- Disallow switching to current tab while still allowing `enterses`

### CODE REVIEW

**Review by erisjacey — COMMENTED** [review]
LGTM. However, would changing `current` to `current session` maybe make the command too long for the user to have to type? Maybe the rest of the team could chime in on this as well. @CodingCookieRookie @YuunoKun @MarcusTw

**Comment by nweiyue** [comment]
> LGTM. However, would changing `current` to `current session` maybe make the command too long for the user to have to type? Maybe the rest of the team could chime in on this as well. @CodingCookieRookie @YuunoKun @MarcusTw

I was thinking about that as well. I am okay with `current` if everyone else is okay with it.

**Comment by CodingCookieRookie** [comment]
> LGTM. However, would changing `current` to `current session` maybe make the command too long for the user to have to type? Maybe the rest of the team could chime in on this as well. @CodingCookieRookie @YuunoKun @MarcusTw

I would prefer current personally :)

File: src/main/java/atas/ui/Tab.java [hunk 1]
```diff
@@ -31,4 +31,29 @@
...
+        return this.equals(Tab.STUDENTS)
+                || this.equals(Tab.SESSIONS)
+                || this.equals(Tab.CURRENT)
+                || this.equals(Tab.MEMO);
+    }
+
+    /**
+     * Returns a String to be displayed to users.
+     *
+     * @return String value of the tab.
```
Comment (marcustw) [inline src/main/java/atas/ui/Tab.java:50]: Perhaps @return Display name of the tab ?

**Review by marcustw — APPROVED** [review]
LGTM

**Review by erisjacey — APPROVED** [review]
LGTM

**Pull request merged**